web/src/js/components/contentviews/HttpMessage.tsx

```diff
@@ -60,6 +65,29 @@ export default function HttpMessage({ flow, message }: HttpMessageProps) {
         }
     }, [content]);
 
+    const handleClickCopyButton = () => {
+        const url = MessageUtils.getContentURL(flow, message, contentView); //without the 'maxLines' parameter, so we can get the full content of the content view
+        setIsFetchingFullContent(true);
+        setIsCopied(true);
```

Comment:
That line should probably come right before the setTimeout, when you've actually copied the data to the clipboard

Reply:
I refactored it a bit and also hid the button in case there is no content